chain/chain/src/runtime/mod.rs

```diff
@@ -782,6 +782,9 @@ impl RuntimeAdapter for NightshadeRuntime {
             / (runtime_config.wasm_config.ext_costs.gas_cost(ExtCosts::storage_write_value_byte)
                 + runtime_config.wasm_config.ext_costs.gas_cost(ExtCosts::storage_read_value_byte));
 
+        // FORKNET: reduce size limit of new transactions to 4MB.
+        let size_limit = (size_limit as f64 / 13.0 * 4.0) as u64;
+
```

Comment:
Let's gate it by some statelessnet protocol feature and also put just 4MB there, what's the reason to compute it based on the old one :)